src/pages/Autorised.js

```diff
@@ -0,0 +1,27 @@
+import { Route, Switch, Redirect } from 'react-router-dom';
+import { TaskList, Task, Main } from '../components';
+import { User, NotFound, Gallery } from '../pages';
+
+
+export const Autorised = ({ user }) => {
+  return (
+    <Switch>
+      <Route path="/" exact component={Main} />
+      <Route
+        path="/home"
+        exact
+        render={props => <Main {...props} user={user} />}
```

Comment:
why you pass there _user_ to Main and don't pass it for `<Route path="/" exact component={Main} />`?